sections/main-password-header.liquid

```diff
@@ -22,7 +22,7 @@
 
     <password-modal>
       <details class="password-modal modal">
-        <summary class="modal__toggle" aria-haspopup="dialog">
+        <summary class="modal__toggle" aria-describedby="a11y-modal-window-message">
```

Comment:
Is this component considered a "modal"? Just asking because in #710, we considered removing `aria-haspopu="dialog"` from this context on `no-js`.